WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -3457,7 +3468,7 @@ private TypeCompability CheckTypeCompatibility(Type? destType, Type? srcType)
         }
 
         if (srcType.IsAssignableTo(typeof(IRedBaseHandle)) &&
-            srcType.GetGenericArguments()[0].IsAssignableTo(destType))
```

Comment:
It should not be changed. Type.IsAssignableTo is true if "The current type is a generic type parameter, and targetType represents one of the constraints of the current type."

Reply:
Is that not exactly the wrong way around? srcType is generic, and destType is not generic.

Reply:
@rfuzzo?